### ISSUE TITLE
php7.0-cli: Dependency gets resolved wrong

### ISSUE THREAD

**Issue by geewiz** [oerdnj/deb.sury.org#372]
After just installing the core packages on a fresh Ubuntu 12.04...

    apt-get install php7.0 php7.0-cli

...the CLI binary throws an error:

    $ php -v
    php: symbol lookup error: php: undefined symbol: pcre_jit_stack_free

At this point, libpcre3 8.12-4 is installed. Its version jumps to `8.38-3.1+deb.sury.org~precise+3` after a manual `apt-get install libpcre3`, and then the CLI works.

**Comment by oerdnj** [comment]
Hmph, this should be really taken care of in src:pcre3. I'll check what I can do about that.

**Comment by geewiz** [comment]
It looks like there's a version constraint missing. The older `libpcre3` comes pre-installed with my Ubuntu box and currently satisfies the dependency in the php7.0 package. But only the newer library package provides `pcre_jit_stack_free`.

**Comment by oerdnj** [comment]
I checked pcre3 and it looks like it's correctly building shlibs version; I'll need to investigate a bit further why '>= 1:83' is not used when building Depends: for php*-*

**Comment by yofisim** [comment]
Any news about this?

**Comment by barrykooij** [comment]
Had the same issue on a clean `12.04` (Precise) box. `apt-get install libpcre3` fixed the issue.

**Comment by Project0** [comment]
Any update on this?

Ubuntu 14.04 same issue with package php7.0-dev:
```
apt-get` install php7.0-dev
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 php7.0-dev : Depends: libpcre3-dev (>= 2:8:20) but 2:8.38-3.1+deb.sury.org~trusty+6 is to be installed
              Recommends: dh-php but it is not going to be installed
              Recommends: pkg-php-tools but it is not going to be installed
E: Unable to correct problems, you have held broken packages.

**Comment by Project0** [comment]
Seems to be fixed with latest release.
https://launchpadlibrarian.net/267483646/php7.0_7.0.8-1+deb.sury.org~trusty+2_7.0.8-2+deb.sury.org~trusty+1.diff.gz

**Comment by oerdnj** [comment]
Yep, libpcre3 dependency and that damn typo fixed in `7.0.8-2` upload.
